po/fr.po

```diff
@@ -2202,7 +2202,7 @@ msgid ""
 "Space separated list of file patterns used for the find in files dialog (e."
 "g. *.c *.h)"
 msgstr ""
-"Liste de motifs de fichier, séparés par une espace, à utiliser dans le "
+"Liste de motifs de fichier, séparés par un espace, à utiliser dans le "
```

Comment:
yep.  But nowadays `un` is taking over, likely because we're not typesetting manually anymore I guess :)